README.md

```diff
@@ -149,7 +149,7 @@ Resources being displayed on the page
 
 
 ## Known Problems
-Currently, if you go to the FAQ page, any user can create a question and answer which will automatically be shown on the page. Any user should not be able to do that, and we need to remove that functionality from the website since it should be part of admin privileges which only OID has. Secondly, if you got to the FAQ page by clicking the button on the header, then click "new FAQ" and click on the FAQ button on the header again, an error is thrown. 
+Currently, the admin privileges provided to the Office of Institutional Diveristy (OID) are limited to editing all mentor profiles, adding new testimonials and adding new FAQs to the website. However, the administrator is not able to delete any mentor profiles, add new resources or update the overview of the program without touching our codebase.
```

Comment:
I like this except can you add the part about the FAQs being able to made from their own page back? That is an issue that we know of

Reply:
wait, sorry what do you mean?